### PR TITLE
v0.5.17

### PR DESCRIPTION
**Pull request by dangodev**
## Checklist

- [x] **CHANGELOG**: The **Unreleased** section of CHANGELOG was updated
- [ ] **Prop changes**: [docs][docs] were updated
- [ ] **Prop changes**: E2E tests were updated, testing from the highest level possible
- [x] **Platform testing**: If this change should be [tested against a platform][platform-testing], has it been?

[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform

### CODE REVIEW

**Comment by dangodev** [comment]
Tested and working in Render Dashboard

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
Comment (davidleger95) [inline CHANGELOG.md]: shoudl the above line be updated to the version number?

Comment (davidleger95) [reply CHANGELOG.md]: beat me to it :P

Comment (dangodev) [reply CHANGELOG.md]: Whoops knew I forgot something 😓

**Pull request merged**